src/MagicOnion.GeneratorCore/CodeAnalysis/MethodCollector.cs

```diff
@@ -26,19 +27,19 @@ public class ReferenceSymbols
 
         public ReferenceSymbols(Compilation compilation, Action<string> logger)
         {
-            Void = compilation.GetTypeByMetadataName("System.Void");
+            Void = compilation.GetTypeByMetadataName("System.Void") ?? compilation.GetSpecialType(SpecialType.System_Void);
             if (Void == null)
             {
                 logger("failed to get metadata of System.Void.");
             }
 
-            TaskOfT = compilation.GetTypeByMetadataName("System.Threading.Tasks.Task`1");
+            TaskOfT = compilation.GetTypeByMetadataName("System.Threading.Tasks.Task`1") ?? GetWellKnownType(94);
```

Comment:
Roslyn's WellKnownTypes sometimes seems to have added enum values.

This means that the int-value may change from version to version, so you will need to use `WellKnownType.GetTypeFromMetadataName` to get the value.
https://github.com/dotnet/roslyn/blob/af17744a8665356567f980b9bcecf23f0817a100/src/Compilers/Core/Portable/WellKnownTypes.cs#L720